### PR TITLE
IGAPP-567: Dont do production releases for aschaffenburg

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by Taggotty — APPROVED** [review]

**Review by SteffiStuffel — APPROVED** [review]

File: .circleci/src/workflows/scheduled_native_promotion.yml [hunk 1]
```diff
@@ -18,12 +18,12 @@ jobs:
       context:
         - mattermost
         - tuerantuer-google-play
-  - promote_android:
-      name: promote_aschaffenburg_android
-      build_config_name: aschaffenburg
-      context:
-        - mattermost
-        - tuerantuer-google-play
+#  - promote_android: # TODO IGAPP 567 Uncomment
```
Comment (maxammann) [inline .circleci/src/workflows/scheduled_native_promotion.yml:21]: IGAPP-567 is the current task, I'm a bit confused :thinking:

**Review by maxammann — APPROVED** [review]
Can't this be done via our environment variables?

File: .circleci/src/workflows/scheduled_native_promotion.yml (hunk 1 above)
Comment (klinzo) [reply .circleci/src/workflows/scheduled_native_promotion.yml:21]: :D we have to disable it first before we can enable it again... I think "Arbeitsbeschaffungsmaßnahme" is the right term for this.

**Comment by klinzo** [comment]
@maxammann it can, but not depending on production/development. So we can only not deliver at all or deliver all the time. This just adds an additional security that we don't accidentally release to produciton before we should

**Comment by maxammann** [comment]
> @maxammann it can, but not depending on production/development. So we can only not deliver at all or deliver all the time. This just adds an additional security that we don't accidentally release to produciton before we should

Cant we just disable the promotion and continue to deliver? e.g. by setting SKIP_BUILD_promote_ios to aschaffenburg.

**Review by maxammann — APPROVED** [review]

**Comment by klinzo** [comment]
Discussed this on mattermost. Disabling promotions does not cover manually triggered production releases and as this are very few changes the benefits of additional security of not releasing aschaffenburg to early is more important.

**Pull request merged**